__tests__/app.test.js

```diff
@@ -57,12 +57,53 @@ describe('GET /api/reviews', () => {
             expect(reviews).toEqual(response.body.reviews);
         })
     });
+    describe('GET /api/reviews/:review_id', () => {
+        test('responds with a review object', () => {
+            return request(app).get('/api/reviews/1').expect(200).then((response) => {
+                expect(response.body.review).toMatchObject({
```

Comment:
As we are only expecting one object back and only want it to be our specifically searched ID, we can be a bit more specific with our expects here. We still want to be using a more flexible matcher (like toMatchObject) but instead of expect.any for review_id, we can just expect it to be 1 - in order to check the path worked as intended.